ruby-push-notifications.gemspec

```diff
@@ -20,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
Ok, so I've just discovered it is a `factory_girl`'s dependency so I assume we should do the build matrix you suggest